app/helpers/template_helpers.py

```diff
@@ -232,73 +316,11 @@ def get_census_base_url(schema_theme: str, language_code: str) -> str:
     return CENSUS_EN_BASE_URL
 
 
-@lru_cache(maxsize=None)
-def get_static_content_urls(
-    language_code: str, base_url: str, schema_theme: str
-) -> Mapping:
-    is_nisra_theme = schema_theme == "census-nisra"
-    if language_code == "cy":
-        help_path = "help/sut-i-ateb-y-cwestiynau/help-y-cwestiynau-ar-lein/"
-        cookies_path = "cwcis/"
-        accessibility_statement_path = "datganiad-hygyrchedd/"
-        privacy_and_data_protection_path = "preifatrwydd-a-diogelu-data/"
-        terms_and_conditions_path = "telerau-ac-amodau/"
-        contact_us = "cysylltu-a-ni/"
-        languages = "help/ieithoedd-a-hygyrchedd/ieithoedd/"
-        bsl_and_audio_videos = (
-            "help/ieithoedd-a-hygyrchedd/hygyrchedd/fideos-hygyrch-gyda-bsl/"
-        )
-    else:
-        help_path = (
-            "help/help-with-the-questions/online-questions-help/"
-            if is_nisra_theme
-            else "help/how-to-answer-questions/online-questions-help/"
-        )
-        cookies_path = "cookies/"
-        accessibility_statement_path = "accessibility-statement/"
-        privacy_and_data_protection_path = "privacy-and-data-protection/"
-        terms_and_conditions_path = "terms-and-conditions/"
-        contact_us = "contact-us/"
-        languages = "help/languages-and-accessibility/languages/"
-        bsl_and_audio_videos = (
-            "help/languages-and-accessibility/accessibility/accessible-videos-with-bsl/"
-        )
-    urls = {
-        "help": f"{base_url}{help_path}",
-        "cookies": f"{base_url}{cookies_path}",
-        "accessibility_statement": f"{base_url}{accessibility_statement_path}",
-        "privacy_and_data_protection": f"{base_url}{privacy_and_data_protection_path}",
-        "terms_and_conditions": f"{base_url}{terms_and_conditions_path}",
-        "contact_us": f"{base_url}{contact_us}",
-    }
-    if not is_nisra_theme:
-        urls["languages"] = f"{base_url}{languages}"
-        urls["bsl_and_audio_videos"] = f"{base_url}{bsl_and_audio_videos}"
-
-    return urls
-
-
-def safe_content(content: str) -> str:
-    """Make content safe.
-
-    Replaces variable with ellipsis and strips any HTML tags.
-
-    :param (str) content: Input string.
-    :returns (str): Modified string.
-    """
-    if content is not None:
-        # Replace piping with ellipsis
-        content = re.sub(r"{.*?}", "…", content)
-        # Strip HTML Tags
-        content = re.sub(r"</?[^>]+>", "", content)
-    return content
-
-
-def get_data_layer(schema_theme: str) -> List:
-    if schema_theme == "census-nisra":
+def get_data_layer(theme: str) -> List:
```

Comment:
It makes sense for it to be set on the context options - it's not used anywhere else apart from the single usage in this module

Reply:
Updated so that it's now set on the ContextOptions class. Note that it needs to be set on the `__post_init__` methods for Census classes so that it overrides the mutable default value of `[]`